lib/Yam/Agama/Pom/AgamaUpAndRunningPage.pm

```diff
@@ -16,7 +16,7 @@ use testapi;
 sub new {
     my ($class, $args) = @_;
     return bless {
-        tag_array_ref_any_first_screen_shown => [qw(agama-product-selection agama-configuring-the-product agama-installing)]
+        tag_array_ref_any_first_screen_shown => [qw(agama-product-selection agama-configuring-the-product agama-installing agama-sle-overview)]
```

Comment:
I think here we need two classes here, kind of Sle::AgamaUpAndRunningPage and Opensuse::AgamaUpAndRunningPage.